projects/shikshalokam/src/lib/core-module/services/config.service.ts

```diff
@@ -0,0 +1,4 @@
+export const BaseConfig = {
+    baseURL: 'https://devhome.shikshalokam.org/',
```

Comment:
It should not be hardcoded. It should come from portal

Reply:
Changed